src/Microsoft.XmlSerializer.Generator/dir.props

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <IsNETCoreApp>true</IsNETCoreApp>
+    <IsNETCoreAppRef>false</IsNETCoreAppRef>
+    <IsUAP>true</IsUAP>
```

Comment:
Yes, I think so (IsUAP=false).

---
In reply to: [109057070](https://github.com/dotnet/corefx/pull/17615#discussion_r109057070) [](ancestors = 109057070)